src/test/java/org/jenkinsci/plugins/ewm/TestUtil.java

```diff
@@ -47,4 +62,28 @@ public int compare(Disk disk1, Disk disk2) {
             }
         });
     }
+
+    public static void addExternalWorkspaceNodeProperty(Node node, String diskPoolRefId, DiskNode... diskNodes) {
```

Comment:
If it's public, it deserves some Javadoc.
Or maybe the entire class could be made Restricted